src/main/java/io/specto/hoverfly/junit/dsl/matchers/PlainTextMatcher.java

```diff
@@ -0,0 +1,6 @@
+package io.specto.hoverfly.junit.dsl.matchers;
+
+public interface PlainTextMatcher extends RequestMatcher {
```

Comment:
I mean inheritance with interfaces. "Marker" interfaces with no methods are almost always an anti-pattern. It should introduce a new method or not exist.